### DISCUSSION TITLE
The chat system that Minecraft has

### DISCUSSION THREAD

**Discussion by myavuzokumus** [Zrips/CMI#6977] [Feature Requests]
Can the chat setting menu, which is integrated into Minecraft and opened with the P key, be integrated into the CMI? In this way, silencing can be done through this menu.
![image](https://user-images.githubusercontent.com/47111280/173897991-a3f54ebc-6f4f-44cc-b4a3-985a99d56f1c.png)

**Comment by mrfloris** [comment]
I think this is probably client-side only? The server (as far as I am aware) does not see if a player ignored another player. 

CMI itself has /cmi ignore, you can enable it in alias.yml, and grant the default group the perm. So if just pressing p + ignore doesn't stop abuse, the player can also use /ignore to further this.
